src/ucs/sys/init.c

```diff
@@ -83,7 +83,9 @@ static UCS_F_NOOPTIMIZE void ucs_check_cpu_flags(void)
     }
 }
 
-static void ucs_modules_load()
+void ucs_modules_load();
+
+void __attribute__((weak)) ucs_modules_load()
```

Comment:
why needed? pls add comment

Reply:
It's needed to disable UCS module loading from VFS (which defines strong symbol `ucs_modules_load`)
Ok, will add a comment